subiquity/common/types.py

```diff
@@ -337,6 +338,12 @@ class ShutdownMode(enum.Enum):
     POWEROFF = enum.auto()
 
 
+class InstallerChannels(CoreChannels):
```

Comment:
Sorry for doing this but can this be in subiquity/server please? common should only be for things that might be imported by the client.

Reply:
Thanks for the reminder